src/Management/test/EndpointCore.Test/Health/EndpointMiddlewareTest.cs

```diff
@@ -163,7 +165,8 @@ public async void HealthActuator_ReturnsMicrosoftHealthDetails()
             var health = JsonSerializer.Deserialize<Dictionary<string, object>>(json);
             Assert.NotNull(health);
             Assert.True(health.ContainsKey("status"));
-            Assert.True(health.ContainsKey("diskSpace"));
+            Assert.True(health.ContainsKey("details"));
```

Comment:
Comment in line 164 is lying now 😉 Structure should be the following:
`{ "status":"UP", "details": { "diskSpace": { "total":499581448192, "free":407577710592, "threshold":10485760, "status": "UP" } } }`